sdk/personalizer/ai-personalizer-rest/tests.yml

```
@@ -0,0 +1,22 @@
+# Parameterize Location to the test resources deployment script. This allows
+# specifying a different Azure region to use when manually triggering the live
+# tests pipeline. This is useful for when we want to test in different
+# environments: Prod, Canary, etc.
+parameters:
+- name: Location
+  displayName: Location
+  type: string
+  default: eastus2euap
+
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure-rest/ai-personalizer"
+      ServiceDirectory: personalizer
+      Location: "${{ parameters.Location }}"
+      EnvVars:
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

Assuming you add a bicep template, we set these env vars automatically now and are deprecating one-off devops variables. But you still need the variable redirect:

```suggestion
        AZURE_CLIENT_ID: $(PERSONALIZER_CLIENT_ID)
        AZURE_TENANT_ID: $(PERSONALIZER_TENANT_ID)
        AZURE_SUBSCRIPTION_ID: $(PERSONALIZER_SUBSCRIPTION_ID)
        AZURE_CLIENT_SECRET: $(PERSONALIZER_CLIENT_SECRET)
```

---

Added these variables.